### DISCUSSION TITLE
OSMap plugin

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [JoomCoder-com/JoomCCK#194] [Q&A]
Trying to rewrite  cobalt xmap plugin to OSMap plugin. But links will not show up. Can you help?

[plg_joomcck.osmap (1).zip](https://github.com/JoomCoder-com/JoomCCK/files/14808036/plg_joomcck.osmap.1.zip)

**Comment by andiamine** [comment]
thanks, we will include it next week and fix all issues in it

**Comment by andiamine** [comment]
Done, was added in extended version, will be available on next update.

**Reply by LordHelmchen666** [reply]
I can not find it in 5.8.0 extended. Did you forgot to add?

**Reply by andiamine** [reply]
Team didnt finish it will be available in next update

**Reply by LordHelmchen666** [reply]
5.9.0 is out, but i can not find OSmap plugin. Where is it?

**Reply by LordHelmchen666** [reply]
Will it ever be released? It is getting annoying to wait always that long and getting no answer. :(

**Comment by LordHelmchen666** [comment]
@andiamine No answer?

**Comment by andiamine** [comment]
Hi @LordHelmchen666  we do apologize for delay, yes was done is coming in next update of JoomCCK Extended
thanks for your patience and support

**Reply by LordHelmchen666** [reply]
Thank you!
